content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Customizing Istio Metrics
+description: This task shows you how to customize the Istio metrics.
+weight: 25
+keywords: [telemetry,metrics,customize]
+---
+
+This task shows you how to customize the metrics that Istio generates.
+
+Istio generates telemetry that various dashboards consume to help you visualize
+your mesh. For example, dashboards that support Istio include:
+
+* [Grafana](/docs/tasks/observability/metrics/using-istio-dashboard/)
+* [Kiali](/docs/tasks/observability/kiali/)
+* [Prometheus](/docs/tasks/observability/metrics/querying-metrics/)
+
+By default, Istio defines and generates a set of standard metrics (e.g.
+`requests_total`), but you can also customize them and create new metrics.
+
+## Custom statistics configuration
+
+Istio uses the Envoy proxy to generate metrics and provides its configuration in
+the `EnvoyFilter` at
+[`manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml`]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml).
+
+Configuring custom statistics involves two sections of the
+`EnvoyFilter`: `definitions` and `metrics`. The `definitions` section
+supports creating new metrics by name, the expected value expression, and the
+metric type (`counter`, `gauge`, and `histogram`). The `metrics` section
+provides values for the metric dimensions as expressions, and allows you to
+remove or override the existing metric dimensions. You can modify the standard
+metrics using `tags_to_remove` or by re-defining a dimension.
+
+## Before you begin
+
+[Install Istio](/docs/setup/) in your cluster and deploy an application.
+Alternatively, you can set up custom statistics as part of the Istio
+installation.
+
+## Enable custom metrics
+
+Edit the the `EnvoyFilter` to add or modify dimensions and metrics. Then, add
+annotations to all the Istio-enabled pods to extract the new or modified
+dimensions.
+
+1. Locate the `envoy.wasm.stats` extension configuration. The default
+   configuration is in the `configuration` section and typically looks like:
+
+    {{< text yaml >}}
+    {
+        "debug": "false",
+        "stat_prefix": "istio"
+    }
+    {{< /text >}}
+
+1. Modify the configuration section for each instance of the extension
+   configuration. For example, to add `destination_port` and `request_host`
+   dimensions to the standard `requests_total` metric, change the configuration
+   section to look like the following. Istio automatically prefixes all metric
+   names with `istio_`, so omit the prefix from the name field in the metric
+   specification.
+
+    {{< text yaml >}}
```

Comment:
Can you write the complete EnvoyFilter instead of just the additional fields and show/before after?